pkg/api/v1/types.go

```diff
@@ -1152,6 +1152,8 @@ type ServiceSpec struct {
 	// users to handle external traffic that arrives at a node.
 	DeprecatedPublicIPs []string `json:"deprecatedPublicIPs,omitempty" description:"deprecated. externally visible IPs (e.g. load balancers) that should be proxied to this service"`
 
+	LoadBalancerIP string `json:"loadBalancerIP, omitEmpty`
```

Comment:
From what I can tell AWS does not have an equivalent way to pre-specify.

@anguslees for a consult on OpenStack